src/screens/main/index.js

```diff
@@ -7,69 +7,143 @@ import { AppText } from "../../components";
-                                <AppText style={{ fontSize: 18, color: Colors.primary }}>{user.name}</AppText>
-                            </View>
-                        </View>
-                    )
+      <MainStack.Screen
+        name={Routes.main.shopScreen}
+        component={IngredientShop}
+        options={({ route, navigation }) => ({
+          headerShown: true,
+          headerTintColor: "white",
+          headerTitle: "Ingredient Shop",
+          headerStyle: {
+            backgroundColor: "black",
+          },
+          header: () => (
+            <LinearGradient
+              start={{ x: 0, y: 1 }}
+              end={{ x: 0, y: 0 }}
+              colors={["transparent", "#0008", "#0009", Colors.seconday]}
+              locations={[0, 0.3, 0.2, 0.9]}
+              style={{
+                width: "100%",
+                padding: 10,
+                paddingTop: 60,
+                flexDirection: "row",
+                justifyContent: "space-between",
+                alignItems: "center",
+              }}
+            >
+              <TouchableOpacity onPress={() => navigation.goBack()}>
+                <IonIcons name="chevron-back" size={28} color="white" />
+              </TouchableOpacity>
+              <AppText style={{ fontSize: 18, textAlign: "center" }}>
+                Ingredient Shop
+              </AppText>
+              <IonIcons name="help-circle" size={28} color="white" />
+            </LinearGradient>
+          ),
+        })}
+      />
 
-                })}
+      <MainStack.Screen
+        name={Routes.main.profileScreen}
+        component={ProfileScreen}
+        options={({ route, navigation }) => ({
+          headerShown: true,
+          headerTintColor: "white",
+          headerTitle: "Ingredient Shop",
+          headerStyle: {
+            backgroundColor: "black",
+          },
+          header: () => (
+            <LinearGradient
+              start={{ x: 0, y: 1 }}
+              end={{ x: 0, y: 0 }}
+              colors={["transparent", "#0008", "#0009", Colors.seconday]}
+              locations={[0, 0.1, 0.2, 0.9]}
+              style={{
+                width: "100%",
+                padding: 10,
+                paddingTop: 60,
+                flexDirection: "row",
+                justifyContent: "space-between",
+                alignItems: "center",
+              }}
+            >
+              <TouchableOpacity onPress={() => navigation.goBack()}>
+                <IonIcons name="chevron-back" size={28} color="white" />
+              </TouchableOpacity>
+              <AppText style={{ fontSize: 18, textAlign: "center" }}>
+                Profile
+              </AppText>
+              <TouchableOpacity onPress={()=>{
+                 dispatch(setUserLogOutState())
+                 navigation.navigate(Routes.onBoarding.welcomeScreen)
```

Comment:
Replace `Routes.onBoarding.welcomeScreen` to `Routes.onBoarding.tag`

Reply:
done